src/UI/state/speciesInformation/actions/upsertSpeciesInfo.action.ts

```diff
@@ -0,0 +1,38 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import { fetchGraphQlDataAuthenticated } from '../../../api/api';
+import {
+  speciesInformationById,
+  upsertSpeciesInformationMutation,
+} from '../../../api/queries';
+import { SpeciesInformation } from '../../state.types';
+import { AppState } from '../../store';
+import { setCurrentInfoForEditing } from '../speciesInformationSlice';
+
+export const upsertSpeciesInformation = createAsyncThunk(
+  'speciesInformation/upsert',
+  async (speciesInformation: SpeciesInformation, { getState }) => {
+    try {
+      const token = (getState() as AppState).auth.token;
+      await fetchGraphQlDataAuthenticated(
+        upsertSpeciesInformationMutation(speciesInformation),
+        token
+      );
+    } catch (e) {
+      console.log(e);
+    }
+  }
+);
+
+export const getSpeciesInformation = createAsyncThunk(
+  'speciesInformation/getWithId',
+  async (id: string, { getState, dispatch }) => {
+    const token = (getState() as AppState).auth.token;
+
+    let res = await fetchGraphQlDataAuthenticated(
```

Comment:
Interesting that you've set this to use the authenticated endpoint - does it need to? I assume we'll reuse this action to get the species information for the information pages, which I don't think need to be auth-ed.

Reply:
You're quite right - this needs to be the unauthenticated route

Reply:
Fixed now